DB/src/main/java/io/deephaven/db/tables/TableDefinition.java

```diff
@@ -71,21 +61,11 @@ public TableDefinition(@NotNull final ColumnDefinition[] columnDefs) {
     }
 
     public TableDefinition(@NotNull final TableDefinition other) {
-        this(other, true);
-    }
-
-    public TableDefinition(@NotNull final TableDefinition other, final boolean setColumns) {
-        super();
-        this.name = other.name;
-        this.namespace = other.namespace;
-        this.storageType = other.storageType;
-        if (setColumns) {
-            this.setColumns(other.columns);
-            this.columnNameMap = other.columnNameMap;
-        }
+        this.setColumns(other.columns);
+        this.columnNameMap = other.columnNameMap;
     }
 
-    public static TableDefinition tableDefinition(@NotNull final Class<?>[] types, @NotNull final String[] columnNames) {
+    public static TableDefinition tableDefinition(@NotNull final Class[] types, @NotNull final String[] columnNames) {
```

Comment:
Why did you drop the `<?>`, thus provoking a new raw types warning?

Reply:
I went on a rampage adding <?> everywhere and then realized mid-way-through that it was going to not-end-well, so started "reverting".  This was likely a case of "reverted too much".